src/main/java/hw5/steps/HomePageSteps.java

```diff
@@ -0,0 +1,197 @@
+package hw5.steps;
+
+import hw5.model.User;
+import hw5.model.UserManager;
+import hw5.page.HomePage;
+import hw5.page.HomePageFrame;
+import hw5.page.LoginForm;
+import hw5.utils.Helper;
+import hw5.utils.PropertiesHandler;
+import io.qameta.allure.Description;
+import io.qameta.allure.Step;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.Properties;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class HomePageSteps {
```

Comment:
What is the purpose of the current class?

Reply:
It was copied from previous hw. I will remove it to make clear location for actual steps.